dockerhub/travis-ci-image.dockerfile

```diff
@@ -49,9 +41,11 @@ ENV LANG en_US.UTF-8
 RUN update-locale
 RUN locale-gen $LANG
 
+RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
```

Comment:
Hm you should have pip already with Ubuntu, why this?

Reply:
No, we need to install pip. It's not packaged in either python3-pip or python3.7-pip (third party), those packages does not exist. You don't get it with python3{.7}-dev either. So actually need to install pip for both 3.6 and 3.7 like this.

Reply:
I can show you, let me push a patch.

Reply:
(we need 3.7 pip as well)

Reply:
We'd get python3.6 pip with python3-pip, but python3.7-pip does not exist in the third party repo. We are testing with 3.7 right now, not sure if we will break 3.7 (no pip?) if we merge this?

Reply:
Let me check, it should be easy to see.